storage/innobase/lock/lock0lock.cc

```diff
@@ -937,7 +986,7 @@ void wsrep_report_error(const lock_t* victim_lock, const trx_t *bf_trx)
   lock_rec_print(stderr, bf_trx->lock.wait_lock, mtr);
   WSREP_ERROR("victim holding lock: ");
   lock_rec_print(stderr, victim_lock, mtr);
-  wsrep_assert_no_bf_bf_wait(victim_lock, bf_trx);
+  wsrep_assert_no_bf_bf_wait(victim_lock, bf_trx, 0);
```

Comment:
The value `0` would seem to correspond to a intention-shared record lock. This is debug code, and such locks should be impossible. In any case, it would be clearer to use the value `LOCK_NONE` here; in fact, you could declare that as a default value of the added function parameter, so that you would not have to change this invocation.

Reply:
No it is not that. Here we do not know what lock type was requested, only conflicting lock is known.